cfitsio3430patch/fitsio.h

```diff
@@ -797,6 +797,7 @@ int CFITS_API fits_delete_iraf_file(const char *filename, int *status);
 /*---------------- utility routines -------------*/
 
 float CFITS_API ffvers(float *version);
+int CFITS_API fits_use_standard_strings(void);
```

Comment:
So here, we prob want to make a short name for this function and then put the long name in longname.h.

Reply:
Maybe, but note there is precedence for long name only.  See the compression routines toward the bottom

Reply:
Also, the short ones are legacy from fortran, and our new feature would not be applicable in fortran where there are no null terminated strings.

Reply:
Ahhhhh cool. Then let's punt.